src/schema/ecommerce/orders.ts

```diff
@@ -73,6 +87,28 @@ export const Orders = {
               }
             }
           }
+          pageCursors {
```

Comment:
i think this can all go into `PageInfo` too so we avoid duplicating it in other places

Reply:
`PageInfo` is also used by the line items, so I decided to put `totalPages` and `pageCursors` in a separate query helper called `PaginationFields`.